lib/elixir/test/elixir/kernel/raise_test.exs

```diff
@@ -57,6 +57,37 @@ defmodule Kernel.RaiseTest do
     end
   end
 
+  test "raise preserves the stacktrace" do
```

Comment:
Move this to the start of the file so there's less chance of line changes?

Reply:
Good call, will do!